sdk/src/Core/Amazon.Runtime/Credentials/FallbackCredentialsFactory.cs

```diff
@@ -59,10 +59,22 @@ public static void Reset(IWebProxy proxy)
             };
         }
 
-        private static AWSCredentials GetAWSCredentials(ICredentialProfileSource source)
+        public static string GetProfileName()
```

Comment:
Does this need to be `public`? Since it is only being used in Core it could be `internal`.

Reply:
Since it was a calculated property made it public, in case it was required outside of Core. But happy to make it internal for now. Pushed up the changes.